modules/apps/27-interchain-accounts/keeper/keeper_test.go

```diff
@@ -21,7 +21,7 @@ var (
 	// TestOwnerAddress defines a reusable bech32 address for testing purposes
 	TestOwnerAddress = "cosmos17dtl0mjt3t77kpuhg2edqzjpszulwhgzuj9ljs"
 	// TestPortID defines a resuable port identifier for testing purposes
-	TestPortID = fmt.Sprintf("%s-0-0-%s", types.VersionPrefix, TestOwnerAddress)
+	TestPortID = fmt.Sprintf("%s|0|0|%s", types.VersionPrefix, TestOwnerAddress)
```

Comment:
should we use generate port id function? Might be good reason to add a `NewPortID` function if we cannot use the generate portID function

Reply:
Yeah, I think we can use `GeneratePortID` here, I'll just discard the error